### ISSUE TITLE
wrong name for two primitive groups

### ISSUE THREAD

**Issue by ThomasBreuer** [gap-packages/primgrp#17]
The first two groups in the following list have apparently a wrong name.

```
gap> AllPrimitiveGroups( Size, 17971200, IsSimpleGroup, true );
#W  AllPrimitiveGroups: Degree restricted to [ 2 .. 4095 ]
[ 2F(4, 2), 2F(4, 2), 2F(4, 2)', 
  <permutation group of size 17971200 with 2 generators> ]
gap> List( last, NrMovedPoints );
[ 1600, 1755, 2304, 2925 ]
```

**Comment by fingolfin** [comment]
Indeed, these three groups are all isomorphic copies of the Tits group. There seems to be a naming clash here:

In `data/gps22.g`, the groups are called `2F(4, 2)'` (the third group in Thomas' list), and `2F(4, 2)` (the overgroup of size 35942400).

In `data/gps17.g`, these twp groups are called `2F(4, 2)` respectively `2F(4, 2):2`. 

In `data/gps15.g`, we find again `2F(4, 2)` as name for the Tits group.

I agree with Thomas that the simple groups should be called `2F(4, 2)'`, which also agrees with the output of `IsomorphismTypeInfoFiniteSimpleGroup`. I'll submit a pull request for this.

**Comment by alex-konovalov** [comment]
Thank you @ThomasBreuer. I've made a new release, which hopefully be included in GAP 4.10.0 distribution.
